packages/app-store/office365calendar/lib/CalendarService.ts

```diff
@@ -432,6 +430,50 @@ export default class Office365CalendarService implements Calendar {
     return response;
   };
 
+  private async apiGraphBatchCallConcurrent(requests: IRequest[]): Promise<IBatchResponse> {
```

Comment:
I haven't dug into the code to verify but I don't think this is the problem. I think it's because we call the `getAvailability` function in a loop outside of this.

Reply:
Ah, I see what you mean. I think you're right